styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -247,6 +255,21 @@ private static boolean isActiveState(RunStateData runStateData) {
     final Schedule schedule;
 
     final WorkflowId workflowId = WorkflowId.create(input.component(), input.workflow());
+
+    final Workflow workflow;
+    try {
+      final Optional<Workflow> workflowResult = storage.workflow(workflowId);
```

Comment:
We read the workflow a few lines down. Why reading it here?

https://github.com/spotify/styx/pull/368/files#diff-0cec30db46b18b2c0132f746e3bad5b0R276